articles/azure-large-instances/workloads/epic/create-a-volume-group.md

```diff
@@ -0,0 +1,99 @@
+---
+title: Create a volume group
```

Comment:
```suggestion
title: Create a volume  groups on a compute host running RHEL (Red Hat Enterprise Linux) 8.4
```
Cc: @akashdubey-ms Lets stick with the same title as from Welcome package to withhold the context